pkgs/tools/misc/websocat/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  name = "websocat-${version}";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner  = "vi";
+    repo   = "websocat";
+    rev    = "refs/tags/v${version}";
```

```suggestion
    rev    = "v${version}";
```

... alternatively an explanatory comment for why the release tarball doesn't work? :)

---

No, I simply always use `refs/tags` -- I was under the impression there's no difference, i.e. that fetchFromGitHub always acquires an archive tarball of some tree snapshot, though I might obviously wrong about that.

---

No, you're right and I'm wrong. Thanks. :)